NEWS

```diff
@@ -8,6 +8,7 @@ Enhancements
  - Appointment color and importance work now between Outlooks
 
 Bug fixes
+ - Fix reception of calendar event invitations on optional attendees
```

Comment:
Showing a color or not may be an enhancement, but being invited to an event (even if set as optional attendee) and not receive any invitation, for me is clearly a bug.